wiremock/applicationStubs.ts

```diff
@@ -1,6 +1,5 @@
 import { guidRegex } from './index'
 
-import applicationSummaryFactory from '../server/testutils/factories/applicationSummary'
```

Comment:
I'm not entirely sure how up to date these stubs are, as we no longer use them in dev, preferring to bring up the whole stack